autopilot/host_test.go

```diff
@@ -0,0 +1,93 @@
+package autopilot
+
+import (
+	"encoding/json"
+	"net"
+	"testing"
+	"time"
+
+	"go.sia.tech/renterd/hostdb"
+	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
+	"lukechampine.com/frand"
+)
+
+func TestHost(t *testing.T) {
```

Comment:
Ran into two definitions of `newTestHost` so decided to move it into `host_test` and by doing so quickly had to cover the `Host` type with some minor unit tests.